Tests/SwiftParserTest/Expressions.swift

```diff
@@ -310,6 +310,36 @@ final class ExpressionTests: XCTestCase {
        """"""#^DIAG^#
        """##
      )
+
+    AssertParse(
+      ##"""
+      #"#^DQ^##^DD^#
```

Comment:
Though I didn't report this input, it was also failed before.

Reply:
I think you can just use a single marker in these tests because the two you have resolve to the same location currently.
```suggestion
      #"#^DIAG^#
```

If you name it `DIAG`, you also don’t need to specify the maker name in `DiagnosticSpec`.

Reply:
I changed to share single `#^DIAG^#`.